### DISCUSSION TITLE
Thumbnails not generated/loaded from s3

### DISCUSSION THREAD

**Discussion by larsjanssen6** [statamic/cms#5956] [General]
Hi, 

My storage container is showing broken thumbnail links pointing to my local domain. I'm using s3 and would expect to see images from there. Any idea how I can fix this? The images are correctly uploaded to the server, and the full images are displayed correctly as well. 

I guess they need to be generated with Glide? Can this be done afterward?

Lars

**Comment by duncanmcclean** [comment]
Have you set `AWS_URL` in your `.env` file?

**Comment by jackmcdade** [comment]
The URL to the filesystem is set either in `config/filesystems.php` or by ENV variables in your .env file. Have you updated the s3 container setting to have the proper URL?
